frontend/src/components/file/DeleteDocument.vue

```diff
@@ -43,10 +43,12 @@ const confirmDelete = (document: Document) => {
 <template>
   <Button
     v-tooltip.bottom="'Delete document'"
+    v-tooltip.focus.bottom="'Delete document'"
```

Comment:
Would prefer all this text be moved to the localization file instead of duplicated.